.travis.yml

```diff
@@ -59,14 +59,14 @@ matrix:
     - env: DB=mysql2 PREPARED_STATEMENTS=true
     - env: DB=mysql2 DRIVER=MariaDB
 
-    - addons:
-        postgresql: "9.4"
```

Comment:
I originally did the addons-postgresql per job instead of global (that one must have slipped in) to save some seconds of setup-time on the non-postgres jobs. It's between 3-4s so it won't matter much